content/modules/ROOT/pages/module-03.adoc

```diff
@@ -0,0 +1,98 @@
+= First steps with Jumpstarter
+
+Jumpstarter is already pre-installed in the developer workspace, you can check for it with
+
+[source,bash]
+----
+jmp version
+----
+
+Jumpstarter interacts with hardware interfaces through "drivers", drivers are written in python to make use of the vast ecosystem of existing python packages. While you can always extend Jumpstarter by writing your own drivers, it comes with a collection of general purpose drivers to help you get started.
+
+[source,bash]
+----
+jmp driver list
+----
+
+TIP: Among the drivers you would notice a few with "mock" in their names, while these mock drivers do not provide access to the hardware, they are useful in showcasing the capability of Jumpstarter
+
+[#config]
+== Configuring Jumpstarter for local development
+
+Create a new file in the workspace named `first-steps.yaml`, this would be an "exporter config file". Within which we would configure a list of drivers representing the hardware interfaces we would like to "export" to the users (that happens to be ourselves in this case).
```

Comment:
```suggestion
Create a new file in the workspace named `first-steps.yaml`, this would be an "exporter config file", in which we will configure a list of drivers representing the hardware interfaces we want to "export" to the users (in this case, ourselves).
```